### PR TITLE
BNH-86 flash message into layout

### PR DESCRIPTION
**Pull request by TomHanmer29**
Moved 2 instances of flashmessage out of individual pages and into layout just once

### CODE REVIEW

**Review by BenRamchandani — CHANGES_REQUESTED** [review]
Couple of things:

* Can you continue to use "FlashType" that the admin page uses (though still display the message if it's null)
* Would be good to replace "ApprovalSuccessMessage" on the profile page too

File: web/Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -77,6 +77,15 @@
...
 </header>
+@{
+    if (TempData["FlashMessage"] != null)
+    {
+        //uses warning as the default alert colour if there's no value in TempData["FlashType"]
+        <div class="alert alert-@(TempData["FlashType"] ?? "warning")" role="alert">
+                @TempData["FlashMessage"]
+        </div>
+    }
+}
```
Comment (MaxTheComputerer) [inline web/Views/Shared/_Layout.cshtml:88]: Could we move this further down somewhere? Maybe in the container or in the `<main>`? I'm not sure I like how it looks when it stretches across the screen 😅

**Review by MaxTheComputerer — COMMENTED** [review]
All looks good with one question 🙂

File: web/Views/Shared/_Layout.cshtml (hunk 1 above)
Comment (TomHanmer29) [reply web/Views/Shared/_Layout.cshtml:88]: Yeah that's a good shout! I'll see if i can put it somewhere slightly less big

**Review by MaxTheComputerer — APPROVED** [review]
Looks great, thanks!

**Review by BenRamchandani — APPROVED** [review]
LGTM. Next time make separate PRs for unrelated changes!

**Pull request merged**